tests/unit/landingai/test_predict.py

```diff
@@ -23,10 +28,38 @@
     UnexpectedRedirectError,
 )
 from landingai.predict import EdgePredictor, Predictor, OcrPredictor
+from landingai.predict.snowflake import SnowflakeNativeAppPredictor
```

Comment:
Why is `SnowflakeNativeAppPredictor` not under `landingai.predict` like the rest??

Reply:
[It is](https://github.com/landing-ai/landingai-python/pull/194/files#diff-d981acd1206798c0fda45b458c72bf6373310bc96892a757aee179ae3a3467ebR4). Auto-import just decided to import it differently.

Let me change it here.